omrmakefiles/rules.linux.mk

```diff
@@ -155,7 +157,7 @@ endif
 ## Debugging Information
 # Indicate that GNU debug symbols are being used
 ifeq (gcc,$(OMR_TOOLCHAIN))
-  ifneq (,$(filter arm ppc s390 x86,$(OMR_HOST_ARCH)))
+  ifneq (,$(filter aarch64, arm ppc s390 x86,$(OMR_HOST_ARCH)))
```

Comment:
Any reason the `aarch64` gets a comma?

Reply:
No that slipped through. Fixed.

Reply:
I think that's a typo which breaks the test.

Reply:
Agreed. Fixed in the latest push.